SignNow.Net/_Internal/Constants/ApiUrl.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+namespace SignNow.Net._Internal.Constants
+{
+    static class ApiUrl
+    {
+        public static Uri ApiBaseUrl = new Uri("https://api.signnow.com");
```

Comment:
for a future - it will be better if we'll use env/config files for this constants